lib/elixir_google_scraper_web/controllers/keyword_controller.ex

```diff
@@ -26,4 +28,18 @@ defmodule ElixirGoogleScraperWeb.KeywordController do
     |> put_flash(flash_type, flash_message)
     |> redirect(to: Routes.keyword_path(conn, :index))
   end
+
+  def show(conn, _) do
+    render(conn, "show.html", keyword: conn.assigns[:keyword])
+  end
+
+  def html(conn, _) do
+    render(conn, "html_response.html", keyword: conn.assigns[:keyword])
+  end
+
+  def set_keyword(conn, _) do
+    keyword = Keywords.get_keyword(conn.params["id"] || conn.params["keyword_id"])
+    conn = assign(conn, :keyword, keyword)
+    conn
+  end
```

Comment:
Maybe easier if we do this

```suggestion
  def show(conn, %{"id" => id}) do
    keyword = Keywords.get_keyword!(id)
    
    render(conn, "show.html", keyword: keyword)
  end

  def html(conn, %{"keyword_id" => keyword_id}) do
     keyword = Keywords.get_keyword!(keyword_id)
     
    render(conn, "html_response.html", keyword: keyword)
  end
```